MultiPage-Clone/index.html

```diff
@@ -1,19 +1,325 @@
+                  ></path>
+                </g>
+              </svg>
+              <span>2035</span>
+            </div>
+            <h1 class="heading">PRICKLES & CO</h1>
+            <p class="hero-card-p">BRING NATURE INDOORS</p>
+            <button class="hero-btn">Shop Now</button>
+          </div>
+        </section>
+        <section class="container">
+          <section class="card-container">
+            <div class="card card-1">
+              <h2 class="card-heading">CACTI</h2>
+              <button class="hero-btn">Shop Collection</button>
+            </div>
+            <div class="card card-2">
+              <h2 class="card-heading">PLANTS</h2>
+              <button class="hero-btn">Shop Collection</button>
+            </div>
+            <div class="card card-3">
+              <h2 class="card-heading">SUCCULENTS</h2>
+              <button class="hero-btn">Shop Collection</button>
+            </div>
+          </section>
+          <section class="columns">
+            <section class="about-us" id="about">
+              <div class="about-us-card">
+                <h2 class="about-us-heading">About US</h2>
+                <p class="about-us-p">
+                  Welcome to Prickles & Co.! We're not just a regular shop; 
+                  we're all about plants and making your home greener.
+                   We've been around since 2035, and we're here to bring nature inside.
+                    Our store is filled with awesome cacti, plants, and succulents to spruce up your space. 
+                    <br />
+                  <br /><br />
 
+                  You might have some questions, and that's cool – check out our FAQs. And if you need to reach us, just hit "Contact." 
+                  The best part? We'll even deliver these lovely plants to your doorstep. So, come on in, explore, and let's have a chat about how 
+                  Prickles & Co. can make your home more lush and vibrant.
+                </p>
+              </div>
+              <div class="/about-us-img-1"></div>
+              <div class="about-us-img-2"></div>
+            </section>
+            <section class="buy-online">
+              <div class="buy-online-img"></div>
+              <div class="buy-online-card">
+                <p class="buy-online-p">BUY ONLINE NOW & GET 10% OFF !</p>
+                <button class="hero-btn buy-online-btn">Shop Now</button>
+              </div>
+            </section>
+          </section>
+        </section>
+        <section class="contact-us" id="contact">
+          <form class="contact-form">
+            <h2 class="about-us-heading">FOR SPECIAL REQUESTS & ORDERS</h2>
```

Comment:
Here is what your box looks like: 
![image](https://github.com/CodeYourFuture/Module-HTML-CSS/assets/118762745/65e49664-47cf-4a74-ab1a-51c35c83cd40)
 
and here is theirs, 
![image](https://github.com/CodeYourFuture/Module-HTML-CSS/assets/118762745/c746ec92-dbfb-479d-b029-bc2df673a025)


Notice how the size and fonts are very different. Also your words dont have a * next to them!